recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -0,0 +1,403 @@
+#!/bin/sh
+
+usage="
+PHYTEC Onboarding Tool for ESEC IoT Device Manager Platform
+For More information: https://osb-cc-esec.github.io
+
+Usage:  $(basename $0) [acceptcontract] [OPTION]
+
+Example:
+        $(basename $0)
+        $(basename $0) acceptcontract --newaccount=<your mailing address>
+        $(basename $0) acceptcontract --onboarding
+
+One of the following options can be selected at a time:
+  acceptcontract                            Accept the contract and no Interface
+  -n, --newaccount=<your mailing address>   New account and Onboarding the device
+                                            for ESEC IoT Device Manager Platform
+  -o, --onboarding                          Only Onboarding the device for ESEC
+                                            IoT Device Manager Platform
+  -h. --help                                This Help
+"
+
+INTERACTIVE=True
+ESECCONTRACT=False
+
+tpm2pkcs11tool='pkcs11-tool --module /usr/lib/libtpm2_pkcs11.so.0'
+CONFIG_PATH="/mnt/config"
+AWSCONFIG_PATH="${CONFIG_PATH}/aws"
+AWSCONFIG_CERT="certs"
+AWSCONFIG_CERTPATH="${AWSCONFIG_PATH}/${AWSCONFIG_CERT}"
+AWSCONFIG_CONFPATH="${AWSCONFIG_PATH}/config"
+ESECCONFIG="esec.config"
+
+TPM_PIN=$(cat /sys/devices/soc0/soc_uid | head -c 7)
+
+calc_wt_size() {
+    # NOTE: it's tempting to redirect stderr to /dev/null, so supress error
+    # output from tput. However in this case, tput detects neither stdout or
+    # stderr is a tty and so only gives default 80, 24 values
+    WT_HEIGHT=18
+    WT_WIDTH=$(tput cols)
```

Comment:
`tput` is not available on the target. You may use `echo $COLUMNS` instead.

Reply:
is changed to WT_WIDTH=$(echo $COLUMNS)